sky/shell/platform_view.h

```diff
@@ -77,11 +77,14 @@ class PlatformView {
 
   std::unique_ptr<Rasterizer> rasterizer_;
   std::unique_ptr<Engine> engine_;
+  SkyEnginePtr engine_ptr_;
```

Comment:
@abarth Good question. Do you have something in mind for how we can make this work uniformly across all platforms? My specific requirement is that I need to be able to call runFromFile (or equivalent) on the engine instance associated with the PlatformView. The PlatformView has an instance of the Engine so, maybe, I could just add a public method and call it directly, but, I think that still won't work for Android because some Java code executes before and after the call to runFromFile.